storybooknative/storybook/stories/CenterView/index.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import * as React from "react";
+import { View } from "react-native";
+import styled from "styled-components";
+
+const Main = styled(View)`
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  background-color: #f5fcff;
+`;
```

Comment:
Also, we could wrap styled like

```
import styled from 'styled-components';

export default (Component: React.Node, style: Object | (() => Object)) =>
  styled(Component)([], style);
```

then we can pass it objects and functions instead of a string

```
import myStyled from './myStyled';

const LolText = myStyled(Text, {
  fontSize: 16,
  color: "pink",
});

const LolText2 = myStyled(Text, props => ({
  color: props.color,
  fontSize: 22,
}));
```

Reply:
True and it looks closer to React Native StyleSheets. The question is: which approach is "better"? I chose to stay as close as possible to the existing standards in this codebase. But this is up for discussion.

Reply:
Well, the type of the style could be our StylePropType instead of Object. That way you know you pass it only valid styles. There is nothing in styled components that let's you know if you typed fnotSize instead of font-size e.g

Reply:
That makes perfect sense. I agree.